x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/async/AsyncSearchIndexServiceTests.java

```diff
@@ -110,7 +110,7 @@ public String toString() {
 
         @Override
         public TestAsyncResponse convertToFailure(Exception exc) {
-            return new TestAsyncResponse(test, expirationTimeMillis, exc.getMessage());
+            return new TestAsyncResponse("", expirationTimeMillis, exc.getMessage());
```

Comment:
Since this is only a test class, could you explain how this improves testing? Is there something wrong with the current implementation that I don't see?